src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -478,14 +474,30 @@ def fuzz(self,
     """LibFuzzerCommon.fuzz override."""
     self._test_ssh()
 
-    #TODO(flowerhack): Pass libfuzzer args (additional_args) here
+    logs.log('SSH works; sync corpus directories')
+    # Push corpus directories to the device.
+    data_dst = self.fuzzer.data_path('corpus')
+    for corpus in corpus_directories:
+      for corpfile in os.listdir(corpus):
+        self.fuzzer.device.store(os.path.join(corpus, corpfile), data_dst)
```

Comment:
TODO before merging: teach `store` to push multiple files at once (e.g. pass a `-r` argument)

Reply:
nvm, it actually already does this via glob